src/cdk/BUILD.bazel

```diff
@@ -16,6 +16,35 @@ ng_module(
     ],
 )
 
+# List of style files that need to be copied to the root of the CDK package. This is
+# necessary in order to keep the Bazel NPM package similar to the previous gulp output.
```

Comment:
Added this information. Though I kept the Gulp backwards-compatibility as the idea is to only have these temporarily, so that we can remove them eventually.